### PR TITLE
Tongyu2

### PR DESCRIPTION
**Pull request by lintongyu418**
* Get hunting service data from the backend

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
 const PREDICTIONS_API_URL = 'http://100.26.159.108:5000';
 
-export { PREDICTIONS_API_URL };
\ No newline at end of file
+const HUNTING_SERVICE_URL = 'http://ec2-54-175-217-176.compute-1.amazonaws.com:8091/hunting/hunting/today';
```
Comment (Lmamaril) [inline src/config.js]: Hi there, can we keep EC2 address in the config file? (found in config.js) @lintongyu418 

Config file: https://github.com/mintso/pnnl-frontend-react/blob/main/src/config.js

You can find an example in importing the url here: https://github.com/mintso/pnnl-frontend-react/blob/main/src/DataAdaptor.js

**Review by Lmamaril — CHANGES_REQUESTED** [review]
Please see comments for ec2 link storage

**Comment by Lmamaril** [comment]
I don't see the Alerts on my local with this branch, is that different for anyone else?

Screen Shot 2021-04-21 at 7 43 45 PM


I see from the API call that the hunting service is not an issue.

**Comment by Lmamaril** [comment]
@lintongyu418 what does it look like from your screen? (add screenshot)

**Comment by Lmamaril** [comment]
I see the issue for me has to do with CORS

**Comment by lttymm** [comment]
> I see the issue for me has to do with CORS

Yes, I have asked Cheng about this and he changed the server setting to allow CORS. I am now waiting on Sabari to redeploy the backend. Once that happens, we should be able to receive the data in front end

**Review by Lmamaril — APPROVED** [review]
Works great for me now and the code looks good!

**Pull request merged**